core/src/test/java/org/pentaho/di/core/database/MySQLDatabaseMetaTest.java

```diff
@@ -38,16 +43,59 @@
 import org.pentaho.di.core.row.value.ValueMetaString;
 import org.pentaho.di.core.row.value.ValueMetaTimestamp;
 
+import java.sql.*;
+
 public class MySQLDatabaseMetaTest {
   MySQLDatabaseMeta nativeMeta, odbcMeta;
 
+  private DatabaseMetaData databaseMetaData;
+
+  private ResultSetMetaData resultSetMetaData, resultSetMetaDataException;
+
+  @Rule
+  public ExpectedException expectedException = ExpectedException.none();
+
   @Before
-  public void setupBefore() {
+  public void setupBefore() throws Exception {

```

Comment:
Please change this method name to just setup. Like in the MariaDB test suite.

Reply:
Hey @pentaho-lmartins: This method has been named like that since before I added the extra methods to the MySQL unit test class. There's a sub-class depending on that method as well, InfiniDbDatabaseMetaTest. I would feel more comfortable leaving it as it is. Let me know your thoughts.